src/tracing_library.cpp

```diff
@@ -151,11 +202,11 @@ NginxVariableFamily TracingLibrary::span_variables() {
   return {.prefix = "datadog_", .resolve = span_property};
 }
 
-std::vector<string_view> TracingLibrary::environment_variable_names() {
-  return {// These environment variable names are taken from `tracer_options.cpp`
-          // and `tracer.cpp` in the `dd-opentracing-cpp` repository.
-          // I did `git grep '"DD_\w\+"' -- src/` in the `dd-opentracing-cpp`
-          // repository.
+std::vector<std::string_view> TracingLibrary::environment_variable_names() {
+  return {// These environment variable names are taken from
+          // `tracer_options.cpp` and `tracer.cpp` in the `dd-opentracing-cpp`
```

Comment:
`environment.h` in the dd-trace-cpp repository?
I guess it's still true where they originally came from and how you did it..